.buildkite/pipeline-build.yml

```diff
@@ -64,8 +64,6 @@ steps:
           machineType: "n1-standard-16"
 
       - label: ":go: helm releaser tool"
-        key: "build-helm-releaser-tool"
         commands:
           - cd hack/helm/release
           - make build
```

Comment:
Yes for the time being. In particular, the hack/helm/release go module depends on eck and after an upgrade of a go dep upgrade in eck, it can fail to compile if we didn't `go mod tidy`.